buildSrc/src/main/kotlin/whitefox.java-conventions.gradle.kts

```diff
@@ -25,6 +28,20 @@ spotless {
         formatAnnotations()
     }
 }
+
+val generatedSources = listOf("io/whitefox/api/model/**", "io/whitefox/api/deltasharing/model/**")
```

Comment:
shouldn't be this defined by the subproject ? like server or client (because they know what they want to exclude, not the root project)